### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by michaelplzno** [mesosoi/silverwaregames-io-issue-tracker#8]
We need a way of making it clear that there is new content for a given user. If they have comments on their posts, or if they have been mentioned etc, they need to have some kind of notification tab.

**Comment by mesosoi** [comment]
Good idea, thank you.

Already had a thought about it. I'm quite a noob speaking of notifications stuff, but I'll try to set it up for as far as I can.

**Comment by mesosoi** [comment]
Yes-yes-yes, I know what notifications are. I was just trying to figure out the best way to implement them. And I guess I found at least a good solution which will be added in the future updates.

**Comment by mesosoi** [comment]
New type of notifications: "Someone just mentioned you".
